hash/ibm5170_cdrom.xml

```diff
@@ -2169,6 +2255,238 @@ license:CC0
+		<info name="usage" value="Windows 95" />
+		<info name="region" value="USA" />
+
+		<part name="cdrom1" interface="cdrom">
+			<diskarea name="cdrom">
+				<disk name="id Anthology (USA) (Disc 1) (Vintage)" sha1="f2e0dcc8252d0b11cc770452f6513fc1d2653452" />
+			</diskarea>
+		</part>
+		<part name="cdrom2" interface="cdrom">
+			<diskarea name="cdrom">
+				<disk name="id Anthology (USA) (Disc 2) (Doom) (Rev 1)" sha1="0cc44f09c53368ac0a5360db9b16d5d1ddf4380b" />
+			</diskarea>
+		</part>
+		<part name="cdrom3" interface="cdrom">
+			<diskarea name="cdrom">
+				<disk name="id Anthology (USA) (Disc 3) (Quake)" status="nodump" />
+			</diskarea>
+		</part>
+		<part name="cdrom4" interface="cdrom">
+			<diskarea name="cdrom">
+				<disk name="id Anthology (USA) (Disc 4) (Mac)" status="nodump" />
+			</diskarea>
+		</part>
+	</software>
+
+	<!-- Source: http://redump.org/disc/28673/ -->
+	<!-- Source: http://redump.org/disc/28674/ -->
+	<!-- <rom name="id Anthology (USA) (Disc 1) (Vintage).cue" size="103" crc="f25a3f47" sha1="c4c9b803f8896960d5e622b71f1ea37202824675"/> -->
+	<!-- <rom name="id Anthology (USA) (Disc 1) (Vintage).bin" size="13034784" crc="59efce1d" sha1="0d68ef00815c2f18817372dacd120e5a2d5ed3c9"/> -->
+	<!-- <rom name="id Anthology (USA) (Disc 2) (Doom).cue" size="123" crc="d9bd95e5" sha1="77f887a39c46b6da72b296fe410b2db46e034362"/> -->
+	<!-- <rom name="id Anthology (USA) (Disc 2) (Doom).bin" size="221391408" crc="7d7d5916" sha1="8652b4195f9796c471880b5150f71022390d9db4"/> -->
+	<software name="idanthola" cloneof= "idanthol">
+		<description>id Anthology (compilation)</description>
+		<year>1996</year>
+		<publisher>id Software</publisher>
+		<notes><![CDATA[
+CD-ROM 1: "Catacomb 3-D", "Commander Keen: Invasion of the Vorticons", "Commander Keen: Goodbye Galaxy", "Commander Keen: Aliens Ate My Babysitter!", "Commander Keen: Keen Dreams",
+		  "Dangerous Dave", "Hover Tank 3D", "Rescue Rover", "Rescue Rover 2", "Shadow Knights", "Slordax: The Unknown Enemy", "Spear of Destiny", "The Catacomb", "Wolfenstein 3D"
+CD-ROM 2: "DOOM II: Hell on Earth", "Final DOOM", "Master Levels for DOOM II", "The Ultimate DOOM"
+]]></notes>
+		<info name="usage" value="DOS" />
+		<info name="region" value="USA" />
+
+		<part name="cdrom1" interface="cdrom">
+			<diskarea name="cdrom">
+				<disk name="id Anthology (USA) (Disc 1) (Vintage)" sha1="f2e0dcc8252d0b11cc770452f6513fc1d2653452" />
+			</diskarea>
+		</part>
+		<part name="cdrom2" interface="cdrom">
+			<diskarea name="cdrom">
+				<disk name="id Anthology (USA) (Disc 2) (Doom)" sha1="b32a547d78a19d2e0a47fdfc64e270b2ecfa000e" />
+			</diskarea>
+		</part>
+		<part name="cdrom3" interface="cdrom">
+			<diskarea name="cdrom">
+				<disk name="id Anthology (USA) (Disc 3) (Quake)" status="nodump" />
+			</diskarea>
+		</part>
+		<part name="cdrom4" interface="cdrom">
+			<diskarea name="cdrom">
+				<disk name="id Anthology (USA) (Disc 4) (Mac)" status="nodump" />
+			</diskarea>
+		</part>
```

Comment:
Maybe include which games are on the Mac disk?

According to https://www.mobygames.com/game/id-anthology, it's
DOOM II: Hell on Earth
Final DOOM
The Ultimate DOOM
Wolfenstein 3D

Reply:
Completed the CD info

Reply:
Looks like there's two entries for the id anthology, so you should probably complete the info for the second entry as well.